src/main/java/com/example/gradecalculator/controller/UserController.java

```diff
@@ -63,12 +75,10 @@ public String userGet(Model model) {
     private String registerPost(Model model, @Valid @ModelAttribute UserSignUpTO registration, BindingResult bindingResult) {
 
         var errors = validateUserSignUpTO(registration, bindingResult);
-
         if (errors.isEmpty()) {
             var user = userService.createUser(registration);
             return "index";
         }
-
```

Comment:
don't quite understand why empty lines are deleted? it is easier to read with some empty lines (especially before/after ifs or methods)

Reply:
i was once told to not have so many empty line, because the code should be short